mlflow/store/sqlalchemy_store.py

```diff
@@ -479,8 +476,9 @@ def _search_runs(self, experiment_ids, filter_string, run_view_type, max_results
                     for exp in experiment_ids
                     for run in self._list_runs(session, exp, run_view_type)]
             filtered = SearchUtils.filter(runs, filter_string)
-            runs = SearchUtils.sort(filtered, order_by)[:max_results]
-            return runs, None
+            sorted_runs = SearchUtils.sort(filtered, order_by)
+            runs, next_page_token = SearchUtils.paginate(sorted_runs, page_token, max_results)
```

Comment:
To be clear, the SQLAlchemy and File stores both use an offset, not a cursor to a specific run for pagination?

Reply:
In this particular implementation, yes. The only difference is if there are runs inserted in the middle in which case there will be subtly different behavior.

Reply:
Got it. That's what I thought